src/Validation/Validation.php

```diff
@@ -908,7 +908,9 @@ public static function range($check, $lower = null, $upper = null)
     public static function url($check, $strict = false)
     {
         static::_populateIp();
-        $alpha = '0-9\p{L}\p{N}';
+
+        $emoji = '\x{1F300}-\x{1F6FF}';
```

Comment:
Should we go higher than U+1F6FF? The [emoji table on wikipedia](https://en.wikipedia.org/wiki/Emoji) has 🧀 at U+1F9C0. The points above 1F6FF are a bit sparse but there are a few good ones up there.

Reply:
Absolutely!